mqtt-discovery.sh

```diff
@@ -203,11 +254,16 @@ function setupHADevicePanelControlExtendedCommands() {
     "name": "'${DEVICE_NAME}'",
     "sw_version": "'${SW_VERSION}'"
    },
-   "name": "Auto Seat & Climate",
-   "payload_press": "auto-seat-and-climate",
+   "icon": "mdi:cctv",
+   "name": "Sentry Mode",
+   "device_class": "switch",
```

Comment:
When I added all the icons, I searched around if it was possible to change the icon for : charger, climate, sentry and steering wheel - because it's a "lightning" icon.

Some would be nicer with a "push button" (round) that turns blue or transparent when off. I am still not sure how to do it... but value_template might be needed.

Reply:
Apparently ... `MQTT Discovery supports “icon” in the JSON payload` but I haven't found a good example.